service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorServiceImpl.java

```diff
@@ -46,18 +42,25 @@ public boolean upsertResourceConnector(final AttributeConnector connector) {
     @Override
     public AttributeConnector retrieveResourceConnector() {
         ZoneEntity zoneEntity = this.zoneResolver.getZoneEntityOrFail();
-        return this.connectorConverter.toConnector(zoneEntity.getResourceAttributeConnector());
+        try {
```

Comment:
Guys, 
This is good we are using Json thanks for fast turnaround but should we define the fields/format of the Json like:
{
  "CLIENTD" : "ddd",
  "CLIENDSECRET" : " secret",
  "UAAURL" : "uaa_url",
  "endpoint" : "ep"
}